tox.ini

```diff
@@ -110,3 +110,28 @@ changedir = {[docs]changedir}
 deps = {[docs]deps}
 commands =
     sphinx-build -W -b linkcheck . {envtmpdir}/linkcheck
+
+[testenv:reppy-py37]
+basepython = python3.7
+deps = 
+    {[testenv:py34]deps}
+    reppy
+
+[testenv:reppy-py27]
+basepython = python2.7
+deps = 
+    {[testenv]deps}
+    reppy
+
+[testenv:rerp-py37]
+basepython = python3.7
+deps = 
+    {[testenv:py34]deps}
+    robotexclusionrulesparser
+
+[testenv:rerp-py27]
+basepython = python2.7
+deps = 
+    {[testenv]deps}
+    robotexclusionrulesparser
```

Comment:
4 new environments look a bit too much to me - what do you think about having two, and installing (or not installing) extra dependencies there, both reppy and robotexclusionrulesparser? It may be not specific to robots.txt, we may have a couple of environments with various optional dependencies installed (or not installed, depending on what we prefer to test more).

Reply:
I have changed the number of environments to 2 (one for Python 3.X and one for Python 2.7).